cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -99,8 +99,11 @@ func addHostFolder(cmd *cobra.Command, args []string) {
 		}
 		if interactive {
 			name, shareType = readNameAndTypeInteractive()
+		}
+		if len(strings.TrimSpace(args[0])) == 0 {
+			atexit.ExitWithMessage(1, usage)
 		} else {
-			name = args[0]
+			name = strings.TrimSpace(args[0])
```

Comment:
@anjannath I suggest to modify the code as below to make it more readable. WDYT?

```diff
diff --git a/cmd/minishift/cmd/hostfolder/add.go b/cmd/minishift/cmd/hostfolder/add.go
index a6eed6b9..21455acc 100644
--- a/cmd/minishift/cmd/hostfolder/add.go
+++ b/cmd/minishift/cmd/hostfolder/add.go
@@ -99,11 +99,11 @@ func addHostFolder(cmd *cobra.Command, args []string) {
                }
                if interactive {
                        name, shareType = readNameAndTypeInteractive()
-               }
-               if len(strings.TrimSpace(args[0])) == 0 {
-                       atexit.ExitWithMessage(1, usage)
                } else {
                        name = strings.TrimSpace(args[0])
+                       if len(strings.TrimSpace(name)) == 0 {
+                               atexit.ExitWithMessage(1, usage)
+                       }
                }
        }
```

code snippet looks like this 

```
if usersShare && runtime.GOOS == "windows" {
		// Windows-only (CIFS), all instances
		name = "Users"
	} else {
		if len(args) < 1 && !interactive {
			atexit.ExitWithMessage(1, usage)
		}
		if interactive {
			name, shareType = readNameAndTypeInteractive()
		} else {
			name = strings.TrimSpace(args[0])
			if len(strings.TrimSpace(name)) == 0 {
				atexit.ExitWithMessage(1, usage)
			}
		}
	}
```

Reply:
@LalatenduMohanty Looks good!